code/__DEFINES/supermatter.dm

```diff
@@ -11,6 +11,7 @@
 #define HEALIUM_HEAT_PENALTY 4
 #define PROTO_NITRATE_HEAT_PENALTY -3
 #define ZAUKER_HEAT_PENALTY 8
+#define ANTINOBLIUM_HEAT_PENALTY 23
```

Comment:
Is an SM with antinoblium possible without having it delaminate?

Reply:
I tried to make an SM that can handle tritium handle it while testing, but it absolutely got owned. I don't even know if it's possible to make it not delam, but antinoblium is the antithesis of hypernoblium, so I'm going to push it to the extremes.

Reply:
fair, maybe add some minor heat resistance and some other features, maybe random anomaly spawn (the sparky ones at low concentration, the more dangerous ones at higher), or something else that makes antinoblium even more interesting

Reply:
I'm planning on also making it lower damage from high power, so if you can somehow make it survive the heat penalty, you can go way above 9GeV (though it would probably delam).

Reply:
nice

Reply:
Actually I might lower heat penalty to something more sane in order to favour going above 9.2GeV an actual possibility, which will make it more unique than just being a turbo tritium fuel.